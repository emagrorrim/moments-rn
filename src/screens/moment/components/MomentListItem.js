import React from 'react';

import { View, Text } from "react-native";

const MomentListItem = (props) => {
  return (
      <View>
        <Text>{props.moment.content}</Text>
      </View>
  );
};

export default MomentListItem;