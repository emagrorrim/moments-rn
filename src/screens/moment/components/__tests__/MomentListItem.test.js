import MomentListItem from '../MomentListItem';

import React from "react";
import renderer from 'react-test-renderer';

describe("MomentListItem", () => {
  test("rendered as snapshot", () => {
    const tree = renderer.create(<MomentListItem moment={{content: "test content"}} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});