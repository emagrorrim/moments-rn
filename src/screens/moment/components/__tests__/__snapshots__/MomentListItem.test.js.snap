// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<View>
  <Text>
    test content
  </Text>
</View>
`;

exports[`MomentListItem  1`] = `
<View>
  <Text>
    test content
  </Text>
</View>
`;

exports[`MomentListItem rendered as snapshot 1`] = `
<View>
  <Text>
    test content
  </Text>
</View>
`;
